_docs_integrate/attribute-values.md

```diff
@@ -705,7 +705,7 @@ The Attribute Values in this chapter can only be used in a [Relationship Attribu
 
 ## Consent
 
-Represents the consent of a person to a specific topic. If you want to obtain a consent, you can send a [`ReadAttributeRequestItem`]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem) with a Consent-[RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) where the owner is the peer.
+Represents the consent of an Identity to a specific topic. To obtain persistent consent from a peer, a [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) can be sent, which contains a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute), whose `owner` is the peer, with [Consent]({% link _docs_integrate/attribute-values.md %}#consent) as `value.@type` within its `attribute` property. For more details, refer to the documentation of the [Request persistent consent of peer]({% link _docs_integrate/request-persistent-consent-of-peer.md %}) scenario.
```

Comment:
`with [Consent]({% link _docs_integrate/attribute-values.md %}#consent)` will link to exactly the same place where the reader ist currently at :D

Reply:
I did it this way so the reader doesn't wonder why we capitalise "Consent" here and not write it in lowercase, so to better distinguish the word "consent" and the RelationshipAttribute value type "Consent", but you're right, it's clear from the context on the one hand and the link doesn't bring the biggest benefit so I can remove it as well xD

Reply:
I think the better option would be to put it in single ticks `

Reply:
I would just capitalize it without ticks. i think that is clear enough for the reader

Reply:
In most places we even use 

```
`"this style"` 
```

to indicate that it's a string. I think that would be the best.

Reply:
IMO we use the pattern mentioned above in many places, so it would be very consistent.

Reply:
From what I've seen, in most places we use single ticks or just capitalize types. Single ticks + quotation marks rather occur in tables than in free texts. However, I don't really have a strong opinion for this particular case, as the documentation in general doesn't seem too consistent regarding formatting, yet.

Reply:
I think there was a discussion about this before and the result was that using the formatting `...` every time a type is used in the text would lead to texts becoming less readable, as this formatting would then have to be used very frequently. This is why the new guides never use the formatting `...` for types, but always for properties of types and strings. Instead, the types are written in CamelCase and also they contain a link the first time they appear in a paragraph.

Reply:
I'm okay with everything except the link :D

Reply:
Relatable :D